user-guide/src/SUMMARY.md

```diff
@@ -28,4 +28,5 @@
   - [Trait contract refinement](verify/traits.md)
   - [Closures](verify/closure.md)
   - [Specification entailments](verify/spec_ent.md)
+  - [Type Models](verify/type-models.md)
```

Comment:
Please also add the link in the file`verify/summary.md`.

Reply:
Done